weather_app.py

```diff
@@ -21,10 +24,26 @@ def display_weather(weather_data):
         print(f"Weather in {weather_data['name']}: {description}")
         print(f"Temperature: {temperature}°C, Humidity: {humidity}%")
 
+def display_forecast(forecast_data):
+    print("5-Day Weather Forecast:")
+    for item in forecast_data['list']:
+        time = item['dt_txt']
+        temp = item['main']['temp']
+        description = item['weather'][0]['description']
+        print(f"{time}: {description}, Temp: {temp}°C")
+
 def main():
-    city_name = input("Enter a city name: ")
-    weather_data = get_weather(city_name)
-    display_weather(weather_data)
+    while True:
+        city_name = input("Enter a city name: ")
+        weather_data = get_weather(city_name)
+        if weather_data:
+            display_weather(weather_data)
+            # Fetch and display 5-day forecast
```

Comment:
wonder if this could become part of the function name? I usually ask that question when I see comments: can the code be more expressive, and reduce my comments usage to only explaining the 'why' (not what) if that is not intuitive from the code